### ISSUE TITLE
Consistent naming/branding/terminology

### ISSUE THREAD

**Issue by ewingjm** [Capgemini/xrm-datamigration#26]
**Is your feature request related to a problem? Please describe.**

The terminology and branding of this repository is a bit inconsistent. For example, we have:

- xrm-datamigration
- CRM Data Migration
- Xrm Data Migration

And if we include the XrmToolBox plug-in repository:

- CDS Data Migrator
- CRM environments (as opposed to CDS/Dataverse environments)

**Describe the solution you'd like**

It would be great if we could update all of the terminology and naming to be consistent. At the time of writing, we should probably be referring to Dataverse where we've got XRM/CDS/CRM. We're starting to see a suite of tools emerging for Power Apps with similar branding on our NuGet organisation -

![image](https://user-images.githubusercontent.com/34515225/101156958-f8dba480-365b-11eb-9451-7cb449b6b2f8.png)

It's debatable if a more appropriate namespace/package title for this would be `Capgemini.PowerApps.DataMigration` (to align with what we've already got) or `Capgemini.Dataverse.DataMigration`. 

**Additional context**

The repository name would probably also need to be updated as part of this.

**Comment by tdashworth** [comment]
I agree there is a lack of name consistency for the project and within our wider collection of tools. I like both suggestions although I'm leaning towards `Capgemini.Dataverse.DataMigration` as the tool is specifically for the Dataverse component of the platform. 

Also, more than happy to provide a matching logo 😀

**Comment by ewingjm** [comment]
Thanks, @tdashworth - I was leaning towards `Capgemini.Dataverse.DataMigration`, too. Good shout on the logo, as well. Check with @realekhor as I believe he did one for the XrmToolBox plug-in.

**Comment by tdashworth** [comment]
Well remembered @ewingjm - there is already a great logo which can be reused here.
